CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing
+
+We are always welcome your contribution!
+
+## How to contribute
+
+### Find the tasks
+
+We use GitHub issues to manage the tasks. Please find the issues you'd like to contribute to it. `welcome contribute` and `easy` are good for the first contribution.
+
+Of course, it would be great to send PullRequest which has no issue! However, please explain the motivation, the cause, the solution and so on in the description of the PR then.
```

Comment:
Of course, we also accept Pull Requests which don't have an issue. However, in this case please make sure you explain the motivation, cause, solution, etc in the PR description.